src/components/alarm/AlarmController.cpp

```diff
@@ -0,0 +1,109 @@
+/*  Copyright (C) 2021 JF, Adam Pigg, Avamander
```

Comment:
Try to make sure the names are those who have contributed to this specific file.

There's also a few newlines missing from the header.

Reply:
OK, fixed now

Reply:
Perfect!